grafana/system-metrics-report.json

```diff
@@ -6832,16 +6832,16 @@
         "query": "elasticsearch",
         "queryValue": "Elasticsearch-system-metrics",
         "refresh": 1,
-        "regex": "/.*kube-burner.*/",
+        "regex": "/.*system-metrics.*/",
         "skipUrlSync": false,
         "type": "datasource"
       },
       {
         "allValue": null,
         "current": {
           "selected": false,
-          "text": "08a23de0-9aaf-559b-871a-9e9fe15cb90e",
-          "value": "08a23de0-9aaf-559b-871a-9e9fe15cb90e"
+          "text": "0bd0e89f-6efb-57e9-953e-e0b14fe74eb4",
+          "value": "0bd0e89f-6efb-57e9-953e-e0b14fe74eb4"
```

Comment:
What do these UUIDs represent?

Reply:
Ignore, that changes in the UI that affect the json file